common/connection.go

```diff
@@ -192,18 +192,18 @@ func (c *Connection) closeSession(sid target.SessionID, tid target.ID) {
 }
 
 func (c *Connection) createSession(info *target.Info) (*Session, error) {
-	c.logger.Debugf("Connection:createSession", "tid:%v bctxid:%v type:%s", info.TargetID, info.BrowserContextID, info.Type)
+	c.logger.Debugf("Connection:createSession", "tid:%v bid::%v type:%s", info.TargetID, info.BrowserContextID, info.Type)
```

Comment:
The reason that I changed back to `bid` is that these lines take a lot of space and become hard to read :(

Reply:
Sure, but it's only 3 more chars, I doubt it makes it hard to read because of that :smile: These logs usually span more than one line, and `ctx` would add a bit more clarity.

It's super minor though, chances are I won't even notice and would go look directly at the source anyway :man_shrugging: